internal/sign/ocpbuild/manager.go

```diff
@@ -0,0 +1,122 @@
+package ocpbuild
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	buildv1 "github.com/openshift/api/build/v1"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/sign"
+	buildutils "github.com/rh-ecosystem-edge/kernel-module-management/internal/utils/build"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/api"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/auth"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/module"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/registry"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/utils"
+)
+
+type manager struct {
+	client          client.Client
+	maker           Maker
+	ocpBuildsHelper buildutils.OpenShiftBuildsHelper
+	authFactory     auth.RegistryAuthGetterFactory
+	registry        registry.Registry
+}
+
+func NewManager(
+	client client.Client,
+	maker Maker,
+	ocpBuildsHelper buildutils.OpenShiftBuildsHelper,
+	authFactory auth.RegistryAuthGetterFactory,
+	registry registry.Registry) sign.SignManager {
+	return &manager{
+		client:          client,
+		maker:           maker,
+		ocpBuildsHelper: ocpBuildsHelper,
+		authFactory:     authFactory,
+		registry:        registry,
+	}
+}
+
+func (bcm *manager) GarbageCollect(ctx context.Context, modName, namespace string, owner metav1.Object) ([]string, error) {
+
+	//Garbage Collection noti (yet) implemented for Build
+	return nil, nil
```

Comment:
@yevgeny-shnaidman As we didn't open any issue for sign jobs not being collected u/s from the hub version.
Would you mind making sure this current implementation is working with the u/s fix that you plan?